.github/workflows/CI-unixish.yml

```diff
@@ -110,3 +110,22 @@ jobs:
       run: |
         make clean
         make -j$(nproc) test selfcheck CXXOPTS="-O2 -g3 -stdlib=libc++ -fsanitize=memory" LDOPTS="-lc++ -fsanitize=memory"
+
+    - name: Run callgrind
+      if: matrix.os == 'ubuntu-24.04'
+      run: |
+        wget https://github.com/danmar/simplecpp/archive/refs/tags/1.5.1.tar.gz
```

Comment:
why don't we use latest HEAD? Does this create some baseline that you use for some comparison?